sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -828,6 +828,8 @@ object GpuOverrides extends Logging {
       sparkSig = (TypeSig.cpuAtomics + TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP +
         TypeSig.UDT).nested())))
 
+  lazy val fileFormats = basicFormats ++ ShimLoader.getSparkShims.getFileFormats
```

Comment:
Maybe you do not need to create a new variable, just append shim file formats to the original `fileFormats`.
```
  lazy val basicFormats: Map[FileFormatType, Map[FileFormatOp, FileFormatChecks]] = Map(
    (CsvFormatType, FileFormatChecks(
     ...
      cudfWrite = TypeSig.none,
      sparkSig = (TypeSig.cpuAtomics + TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP +
        TypeSig.UDT).nested()))) ++ ShimLoader.getSparkShims.getFileFormats
```

Reply:
The new line makes the code more readable, I think.